README.md

```diff
@@ -48,6 +48,7 @@ Current requirements are:
 ```shell
 git clone https://github.com/NoiseByNorthwest/php-spx.git
 cd php-spx
+git checkout tags/v0.4.10
```

Comment:
```suggestion
git checkout release/latest
```
In order to not have to touch the readme each time I make a new tag (as explained in my previous comment)